src/main/java/uk/gov/pay/api/resources/PaymentsResource.java

```diff
@@ -273,9 +273,9 @@ public Response searchPayments(@ApiParam(value = "accountId", hidden = true) @Pa
                                    @ApiParam(value = "Last four digits of the card used to make payment", hidden = false)
                                    @Parameter(description = "Last four digits of the card used to make payment", hidden = false)
                                    @QueryParam("last_digits_card_number") String lastDigitsCardNumber,
-                                   @ApiParam(hidden = true)
+                                   @ApiParam("From settled date of payout to be searched (this date is inclusive). Example=2015-08-13")
```

Comment:
is it `From settled date of transaction to be searched` ? and all other uses below